ui/src/features/Editor/components/BottomPanel/index.tsx

```diff
@@ -18,7 +18,7 @@ type Props = {
   isOpen: boolean;
   onOpen: (panel?: "left" | "right" | "bottom") => void;
   onWorkflowClose: (workflowId: string) => void;
-  onWorkflowAdd: () => void;
+  //onWorkflowAdd: () => void;
```

Comment:
Remember to remove commented out code 🙏🏻 
If you can look at your code quickly (review it yourself or check CodeRabbit's review at least) you'll be able to clean up your implementation before anyone else reviews 👍🏻